cita-chain/types/src/block_tag.rs

```diff
@@ -0,0 +1,25 @@
+use cita_types::H256;
+use jsonrpc_types::rpc_types::{BlockNumber as RpcBlockNumber, BlockTag as RpcBlockTag};
+
+pub type TransactionHash = H256;
+pub type BlockNumber = u64;
+
+#[derive(Debug, PartialEq, Copy, Clone, Hash, Eq)]
+pub enum BlockTag {
+    Hash(H256),
+    Number(BlockNumber),
+    Earliest,
+    Latest,
+    Pending,
+}
+
+impl From<RpcBlockNumber> for BlockTag {
+    fn from(v: RpcBlockNumber) -> BlockTag {
+        match v {
```

Comment:
what means for `v` ?

Reply:
OK, it should be `n` or `rpc_block_number`.